Cargo.toml

```diff
@@ -10,33 +10,34 @@ repository = "https://github.com/coreos/zincati"
 edition = "2018"
 
 [dependencies]
-actix = "0.10"
```

Comment:
Looks like a bad rebase here.

Reply:
@travier Just rectified it. Could you please take another look?